pages/concepts/fetch-network/intro-fetch-network.mdx

```diff
@@ -0,0 +1,39 @@
+# The Fetch Network
+
+## Introduction
+
+The **Fetch Network** represents the foundation of the Fetch.ai Ecosystem as it enables the decentralized digital economy driven by [Agents ↗️](/guides/agents/getting-started/whats-an-agent), [AI Engine ↗️](/concepts/ai-engine/ai-engine-intro), [Agentverse ↗️](/concepts/agent-services/agentverse-intro) and [DeltaV ↗️](/concepts/ai-engine/deltav). The Network provides integration with Fetch blockchain technology and ensures security, transparency, and scalability of transactions, and enhanced trust across the overall Ecosystem.
+
+### Agents, Agentverse, AI Engine and DeltaV
```

Comment:
Not sure about this, think this introduction should be more focused on token, ledger and web3 things

Reply:
I agree. I just added it as a starting point but was not really sure. I will remove it straightaway.